src/Common/src/Interop/Unix/libc/Interop.opendir.cs

```diff
@@ -9,6 +9,6 @@ internal static partial class Interop
     internal static partial class libc
     {
         [DllImport(Libraries.Libc, SetLastError = true)]
-        internal static extern SafeDirHandle opendir(string name);
+        internal static extern Microsoft.Win32.SafeHandles.SafeDirHandle opendir(string name);
```

Comment:
Good idea. Only ReadDir uses opendir and closedir so I'll add them to the readdir file